### PR TITLE
Pcapminer v2 post fix

### PR DESCRIPTION
**Pull request by orlichter1**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/18521

## Description
A new script to analyze pcap data. 

## Minimum version of Demisto
- [ ] 4.5.0
- [x] 5.0.0
- [ ] 5.5.0

## Does it break backward compatibility?
   - [x] No

## Must have
- [x] Tests
- [ ] [YML](https://github.com/demisto/content/edit/PcapminerV2_post_fix/Packs/PcapAnalysis/Scripts/PcapMinerV2/PcapMinerV2.yml?pr=%2Fdemisto%2Fcontent%2Fpull%2F7150):

### CODE REVIEW

**Comment by orlichter1** [comment]
@yaakovi Done except getting RSA key to work on Demisto (works locally) which I'll be happy to talk with you about on Sunday. @David-BMS is doing QA on the script and the current version has already gone through changes following David's tests.

**Comment by orlichter1** [comment]
@kirbles19 could you please go over the yml? Thanks

File: Packs/PcapAnalysis/pack_metadata.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+    "name": "Pcap Analysis",
```
Comment (michalgold) [inline Packs/PcapAnalysis/pack_metadata.json:2]: please change to "PCAP" here and everywhere else

**Comment by kirbles19** [comment]
@orlichter1 Done.

**Comment by trorabaugh** [comment]
YIPPEEEEEEE let me know when i can test it out!

**Comment by David-BMS** [comment]
@trorabaugh the script itself is ready i'll send you.

**Pull request merged**